security_check/main.py

```diff
@@ -39,6 +39,19 @@
     _CRITICAL: 3,
 }
 
+_SUB_MAP = {
+    'launcher.gcr.io/google': 'gcr.io/google-appengine',
+    'l.gcr.io/google': 'gcr.io/google-appengine'
+}
+
+
+def _sub_image(image):
+    for k in _SUB_MAP.keys():
+        if k in image:
+            image = image.replace(k, _SUB_MAP.get(k))
+
+    return image
```

Comment:
Ah, this is more annoying than I thought it would be since we don't parse the image into a registry and a repo :(

Do you think that would be easier? Basically split on / and the first two parts are the registry/repo then you can combine them, and do a simple lookup?